### PR TITLE
Fix golint issues in pkg/kubelet/checkpointmanager/testing/example_checkpoint_formats/v1

### PR DESCRIPTION
**Pull request by sshukun**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by sshukun** [comment]
/assign @apelisse @vishh
Please take a look, thanks.

File: staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/fields.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 
 // EmptyFields represents a set with no paths
 // It looks like metav1.Fields{Raw: []byte("{}")}
-var EmptyFields metav1.FieldsV1 = func() metav1.FieldsV1 {
+var EmptyFields = func() metav1.FieldsV1 {
```
Comment (apelisse) [inline staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/fields.go]: I can't see a corresponding line in `.golint_failures`, are there many other errors in this package?

**Comment by apelisse** [comment]
/ok-to-test

Looks good to me. I'm not sure the `fieldmanager` change belongs in that PR, but I think this change is great and would really appreciate if some other packages are fixed as well (in different PRs).

File: staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/fields.go (hunk 1 above)
Comment (sshukun) [reply staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/fields.go]: Yes, this package is not contained in `.golint_failures`, but `verify-golint.sh` failed in this file and this is the only error in this package, so I fixed this also. 
I will separate this fix in different PRs.

**Comment by sshukun** [comment]
@apelisse 
Sorry, I am not sure if I am doing right thing here, so I closed this PR and create a new one to separate `fieldmanager` package.

**Comment by apelisse** [comment]
Yeah, I don't think this is the shortest path to where you want to go :-), but I'm sure you'll get there.
Let me know if you want me to look at these PRs, thanks!

**Pull request closed without merging**